eng/CredScanSuppression.json

```diff
@@ -35,9 +35,13 @@
         {
             "file":[
                 "sdk/keyvault/azure-keyvault-certificates/tests/ca.key",
-                "sdk/identity/azure-identity/tests/ec-certificate.pem"
+                "sdk/identity/azure-identity/tests/ec-certificate.pem",
+                "sdk/core/azure-servicemanagement-legacy/tests/legacy_mgmt_settings_fake.py",
```

Comment:
We want to be careful about excluding entire files because that means if there ever becomes an individual secret in one of them that we will not detect it. We are trying to only exclude files if the entire file is a fake key/certificate that is used for testing.

Reply:
I have checked this file. The purpose of this file is to add fake key in common place and use variables in tests. If people do not abuse the file with real key, then it is supposed to suppress in this way.

Reply:
OK yeah I think those cases are also reasonable. The approach also might be interesting for other languages if they need something similar.